contracts/azorius/LinearERC20Voting.sol

```diff
@@ -110,7 +110,7 @@ contract LinearERC20Voting is BaseStrategy, BaseQuorumPercent, BaseVotingBasisPe
     }
 
     /** @inheritdoc BaseStrategy*/
-    function initializeProposal(bytes memory _data) external virtual override onlyAzorius {
+    function initializeProposal(bytes memory _data) public virtual override onlyAzorius {
```

Comment:
maybe we should move this down above `hasVoted` now that it's public, the preserve the external / public ordering